CompuRacer_Core/src/compu_racer_core.py

```diff
@@ -221,9 +224,7 @@ def start(self):
         self.print_formatted("Starting command processor..", utils.QType.INFORMATION)
         time.sleep(0.25)
         utils.clear_output()
-        self.command_processor.start()
-
-
+        self.command_processor.start(cli_check, racer, self.state)
```

Comment:
is 'racer' hier niet gelijk aan 'self'? Je zit tenslotte in de CompuRacer class zelf